### ISSUE TITLE
执行到这里 就不动了

### ISSUE THREAD

**Issue by one-matrix** [index-tts/index-tts#539]
`
INFO:     Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
2025-11-06 22:35:30 - app.routers.tts - INFO - 合成语音: 你好，这是一段测试文本。...
INFO:app.routers.tts:合成语音: 你好，这是一段测试文本。...
2025-11-06 22:35:30 - app.services.tts_service - INFO - Initializing IndexTTS model...
INFO:app.services.tts_service:Initializing IndexTTS model...
IndexTTS model loaded from /root/workspace/deepspeech/checkpoints
>> GPT weights restored from: /root/workspace/deepspeech/checkpoints/gpt.pth
WARNING:indextts.gpt.transformers_modeling_utils:GPT2InferenceModel has generative capabilities, as `prepare_inputs_for_generation` is explicitly overwritten. However, it doesn't directly inherit from `GenerationMixin`. From 👉v4.50👈 onwards, `PreTrainedModel` will NOT inherit from `GenerationMixin`, and this model will lose the ability to call `generate` and other related functions.
  - If you're using `trust_remote_code=True`, you can get rid of this warning by loading the model with an auto class. See https://huggingface.co/docs/transformers/en/model_doc/auto#auto-classes
  - If you are the owner of the model architecture code, please modify your model class such that it inherits from `GenerationMixin` (after `PreTrainedModel`, otherwise you'll get an exception).
  - If you are not the owner of the model architecture class, please contact the model code owner to update it.
>> semantic_codec weights restored from: ./checkpoints/hf_cache/models--amphion--MaskGCT/snapshots/265c6cef07625665d0c28d2faafb1415562379dc/semantic_codec/model.safetensors
cfm loaded
length_regulator loaded
gpt_layer loaded
>> s2mel weights restored from: /root/workspace/deepspeech/checkpoints/s2mel.pth
>> campplus_model weights restored from: ./checkpoints/hf_cache/models--funasr--campplus/snapshots/fb71fe990cbf6031ae6987a2d76fe64f94377b7e/campplus_cn_common.bin
Loading weights from nvidia/bigvgan_v2_22khz_80band_256x
Removing weight norm...
>> bigvgan weights restored from: nvidia/bigvgan_v2_22khz_80band_256x
`

`Package                                  Version
---------------------------------------- -------------
absl-py                                  2.3.1
accelerate                               1.11.0
aiofiles                                 23.2.1
aiohappyeyeballs                         2.6.1
aiohttp                                  3.12.13
aiosignal                                1.4.0
airportsdata                             20250622
aliyun-python-sdk-core                   2.16.0
aliyun-python-sdk-kms                    2.16.5
annotated-types                          0.7.0
antlr4-python3-runtime                   4.9.3
anyio                                    4.9.0
argbind                                  0.3.9
astor                                    0.8.1
asttokens                                3.0.0
attrs                                    25.3.0
audioread                                3.0.1
bcrypt                                   4.3.0
beautifulsoup4                           4.13.4
blake3                                   1.0.5
cachetools                               5.5.2
certifi                                  2025.6.15
cffi                                     1.17.1
charset-normalizer                       3.4.2
click                                    8.1.8
cloudpickle                              3.1.1
cn2an                                    0.5.23
coloredlogs                              15.0.1
compressed-tensors                       0.10.1
conformer                                0.3.2
contourpy                                1.3.2
crcmod                                   1.7
cryptography                             45.0.5
cupy-cuda12x                             13.4.1
cycler                                   0.12.1
cython                                   3.1.2
decorator                                5.2.1
deepspeed                                0.17.1
deprecated                               1.2.18
depyf                                    0.18.0
descript-audiotools                      0.7.2
diffusers                                0.29.0
dill                                     0.4.0
diskcache                                5.6.3
distance                                 0.1.3
distro                                   1.9.0
dnspython                                2.7.0
docstring-parser                         0.17.0
ecdsa                                    0.19.1
editdistance                             0.8.1
einops                                   0.8.1
email-validator                          2.2.0
executing                                2.2.1
fastapi                                  0.115.14
fastapi-cli                              0.0.7
fastrlock                                0.8.3
ffmpeg-python                            0.2.0
ffmpy                                    0.6.4
filelock                                 3.18.0
fire                                     0.7.1
flatbuffers                              25.2.10
flatten-dict                             0.4.2
fonttools                                4.58.5
frozenlist                               1.7.0
fsspec                                   2024.12.0
funasr                                   1.2.6
future                                   1.0.0
g2p-en                                   2.1.0
gdown                                    5.1.0
gguf                                     0.17.1
google-auth                              2.40.3
google-auth-oauthlib                     1.0.0
googleapis-common-protos                 1.70.0
grpcio                                   1.73.1
grpcio-tools                             1.62.3
gtts                                     2.4.0
h11                                      0.16.0
h5py                                     3.15.1
hf-xet                                   1.1.5
hjson                                    3.1.0
httpcore                                 1.0.9
httptools                                0.6.4
httpx                                    0.28.1
huggingface-hub                          0.33.2
humanfriendly                            10.0
hydra-core                               1.3.2
hyperpyyaml                              1.2.2
idna                                     3.10
importlib-metadata                       8.4.0
importlib-resources                      6.5.2
inflect                                  7.3.1
iniconfig                                2.1.0
interegular                              0.3.3
ipython                                  9.7.0
ipython-pygments-lexers                  1.1.1
jaconv                                   0.4.0
jamo                                     0.4.1
jedi                                     0.19.2
jieba                                    0.42.1
jinja2                                   3.1.6
jiter                                    0.10.0
jmespath                                 0.10.0
joblib                                   1.5.1
json5                                    0.12.1
jsonschema                               4.24.0
jsonschema-specifications                2025.4.1
julius                                   0.2.7
kaldifst                                 1.7.14
kaldiio                                  2.18.1
keras                                    3.12.0
kiwisolver                               1.4.8
lark                                     1.2.2
lazy-loader                              0.4
librosa                                  0.10.1
lightning                                2.2.4
lightning-utilities                      0.14.3
llguidance                               0.7.30
llvmlite                                 0.44.0
lm-format-enforcer                       0.10.11
markdown                                 3.8.2
markdown-it-py                           3.0.0
markdown2                                2.5.4
markupsafe                               3.0.2
matplotlib                               3.7.5
matplotlib-inline                        0.2.1
mdurl                                    0.1.2
mistral-common                           1.6.3
ml-dtypes                                0.5.3
modelscope                               1.20.0
more-itertools                           10.7.0
mpmath                                   1.3.0
msgpack                                  1.1.1
msgspec                                  0.19.0
multidict                                6.6.3
munch                                    4.0.0
namex                                    0.1.0
nest-asyncio                             1.6.0
networkx                                 3.5
ninja                                    1.11.1.4
nltk                                     3.9.1
numba                                    0.61.2
numpy                                    1.26.4
nvidia-cublas-cu12                       12.6.4.1
nvidia-cuda-cupti-cu12                   12.6.80
nvidia-cuda-nvrtc-cu12                   12.6.77
nvidia-cuda-runtime-cu12                 12.6.77
nvidia-cudnn-cu12                        9.5.1.17
nvidia-cufft-cu12                        11.3.0.4
nvidia-cufile-cu12                       1.11.1.6
nvidia-curand-cu12                       10.3.7.77
nvidia-cusolver-cu12                     11.7.1.2
nvidia-cusparse-cu12                     12.5.4.2
nvidia-cusparselt-cu12                   0.6.3
nvidia-nccl-cu12                         2.26.2
nvidia-nvjitlink-cu12                    12.6.85
nvidia-nvtx-cu12                         12.6.77
oauthlib                                 3.3.1
omegaconf                                2.3.0
onnx                                     1.17.0
onnxruntime-gpu                          1.22.0
openai                                   1.93.0
openai-whisper                           20250625
opencv-python                            4.11.0.86
opencv-python-headless                   4.11.0.86
opentelemetry-api                        1.27.0
opentelemetry-exporter-otlp              1.27.0
opentelemetry-exporter-otlp-proto-common 1.27.0
opentelemetry-exporter-otlp-proto-grpc   1.27.0
opentelemetry-exporter-otlp-proto-http   1.27.0
opentelemetry-proto                      1.27.0
opentelemetry-sdk                        1.27.0
opentelemetry-semantic-conventions       0.48b0
opentelemetry-semantic-conventions-ai    0.4.10
optree                                   0.17.0
oss2                                     2.19.1
outlines                                 0.1.11
outlines-core                            0.1.26
packaging                                24.2
pandas                                   2.3.3
parso                                    0.8.5
partial-json-parser                      0.2.1.1.post6
passlib                                  1.7.4
pexpect                                  4.9.0
pillow                                   11.3.0
platformdirs                             4.3.8
pluggy                                   1.6.0
pooch                                    1.8.2
proces                                   0.1.7
prometheus-client                        0.22.1
prometheus-fastapi-instrumentator        7.1.0
prompt-toolkit                           3.0.52
propcache                                0.3.2
protobuf                                 3.19.6
psutil                                   7.0.0
ptyprocess                               0.7.0
pure-eval                                0.2.3
py-cpuinfo                               9.0.0
pyarrow                                  18.1.0
pyasn1                                   0.6.1
pyasn1-modules                           0.4.2
pycountry                                24.6.1
pycparser                                2.22
pycryptodome                             3.23.0
pydantic                                 2.11.7
pydantic-core                            2.33.2
pydantic-settings                        2.10.1
pygments                                 2.19.2
pyloudnorm                               0.1.1
pynini                                   2.1.6
pynndescent                              0.5.13
pyparsing                                3.2.3
pyphen                                   0.17.2
pysocks                                  1.7.1
pystoi                                   0.4.1
pytest                                   7.4.3
pytest-asyncio                           0.21.1
python-dateutil                          2.9.0.post0
python-dotenv                            1.1.1
python-jose                              3.3.0
python-json-logger                       3.3.0
python-multipart                         0.0.20
pytorch-lightning                        2.5.2
pytorch-wpe                              0.0.1
pyttsx3                                  2.90
pytz                                     2025.2
pyworld                                  0.3.4
pyyaml                                   6.0.2
pyzmq                                    27.0.0
randomname                               0.2.1
ray                                      2.47.1
referencing                              0.36.2
regex                                    2024.11.6
requests                                 2.31.0
requests-oauthlib                        2.0.0
rich                                     13.7.1
rich-toolkit                             0.14.8
rpds-py                                  0.26.0
rsa                                      4.9.1
ruamel-yaml                              0.18.14
ruamel-yaml-clib                         0.2.12
safetensors                              0.5.3
scikit-learn                             1.7.0
scipy                                    1.11.4
semantic-version                         2.10.0
sentencepiece                            0.2.0
setuptools                               79.0.1
setuptools-rust                          1.11.1
shellingham                              1.5.4
six                                      1.17.0
sniffio                                  1.3.1
soundfile                                0.12.1
soupsieve                                2.7
soxr                                     0.5.0.post1
stack-data                               0.6.3
starlette                                0.46.2
sympy                                    1.14.0
tensorboard                              2.14.0
tensorboard-data-server                  0.7.2
tensorboardx                             2.6.4
tensorrt-cu12                            10.12.0.36
tensorrt-cu12-bindings                   10.12.0.36
tensorrt-cu12-libs                       10.12.0.36
termcolor                                3.2.0
text-normalization                       0.0.1
textstat                                 0.7.11
threadpoolctl                            3.6.0
tiktoken                                 0.9.0
tokenizers                               0.21.2
torch                                    2.7.0
torch-complex                            0.4.4
torch-stoi                               0.2.3
torchaudio                               2.7.0
torchmetrics                             1.7.4
torchvision                              0.22.0
tqdm                                     4.67.1
traitlets                                5.14.3
transformers                             4.52.1
triton                                   3.3.0
typeguard                                4.4.4
typer                                    0.16.0
typing-extensions                        4.14.1
typing-inspection                        0.4.1
tzdata                                   2025.2
umap-learn                               0.5.9.post2
unidecode                                1.4.0
urllib3                                  2.5.0
uvicorn                                  0.35.0
uvloop                                   0.21.0
vllm                                     0.9.1
watchfiles                               1.1.0
wcwidth                                  0.2.14
websockets                               15.0.1
werkzeug                                 3.1.3
wetext                                   0.0.4
wetextprocessing                         1.0.4.1
wget                                     3.2
wheel                                    0.45.1
wrapt                                    1.17.2
xformers                                 0.0.30
xgrammar                                 0.1.19
yarl                                     1.20.1
zipp                                     3.23.0
`
